SpatialGDK/Source/SpatialGDKEditor/Private/Utils/LaunchConfigEditor.cpp

```diff
@@ -25,6 +53,11 @@ void ULaunchConfigurationEditor::SaveConfiguration()
 
 	if (bSaved && Filenames.Num() > 0)
 	{
-		GenerateDefaultLaunchConfig(Filenames[0], &LaunchConfiguration);
+		if (ValidateGeneratedLaunchConfig(LaunchConfiguration, LaunchConfiguration.ServerWorkersMap))
+		{
+			GenerateDefaultLaunchConfig(Filenames[0], &LaunchConfiguration, LaunchConfiguration.ServerWorkersMap);
```

Comment:
Wonder if we should change the name of `GenerateDefaultLaunchConfig` to `GenerateLaunchConfig` which takes in a name var.